src/composables/accounts.ts

```diff
@@ -222,12 +249,83 @@ export function useAccounts() {
     setActiveAccountByProtocolAndIdx(protocol, account?.idx || 0);
   }
 
-  function setMnemonic(newMnemonic: string) {
+  /**
+   * Setting/Resetting the password key logs the user in/out.
+   */
+  function setPasswordKey(key: IKey | null) {
```

Comment:
Argument name `key` is little bit misleading as it has the `key` property so we are ending with `key.key` here.